plugin/storage/integration/elasticsearch_test.go

```diff
@@ -145,9 +161,14 @@ func (s *ESStorageIntegration) initSpanstore(t *testing.T, allTagsAsFields bool)
 }
 
 func healthCheck() error {
+	t := &http.Transport{}
+	defer t.CloseIdleConnections()
+	c := http.Client{
+		Transport: t,
+	}
```

Comment:
let's extract this into `newHTTPClient(t)` and reuse everywhere. For v8 where you only need the transport you can still call this helper and then just use its client.Transport field